README.md

```diff
@@ -111,24 +111,16 @@ config :vintage_net_wizard,
   port: 4001
 ```
 
-### SSL
-
-To use SSL with the web UI, simply set the configuration flag:
+If SSL is enabled the default port is `443`.
 
-```elixir
-config :vintage_net_wizard, ssl: true
-```
+### SSL
 
-This will default to use a self-signed certificate and key on port `443`.
-You can also specify your own certificate, key, and port in the config:
+To use SSL with the web UI, you can pass these options to
+`VintageNetWizard.run_wizard`:
 
-```elixir
-config :vintage_net_wizard,
-  ssl: true,
-  certfile: "path/to/cert.pem",
-  keyfile: "path/to/key.pem",
-  port: 4443
-```
+* `:ssl` - a boolean that will tell the server to use SSL or not (default false)
```

Comment:
I wonder if this might be a little redundant? What if we follow the pattern of using ssl elsewhere and make this a key. If the `:ssl` is there, then we know to use ssl?

```elixir
ssl: [certfile: "/some/path", keyfile: "/some/path"]
```